lib/agent0/agent0/hyperdrive/policies/zoo.py

```diff
@@ -15,6 +19,7 @@ class Zoo(NamedTuple):
     random = Random
     arbitrage = Arbitrage
     smart_long = SmartLong
+    LPandArb = LPandArb
```

Comment:
i think you want the name to be python friendly, you're overriding the import package this way. e.g. `lp_and_arb = LpandArb`